jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/python/PythonDebugUtils.kt

```diff
@@ -33,6 +37,9 @@ object PythonDebugUtils {
                 )
             )
 
+            val sdk = ProjectRootManager.getInstance(environment.project).projectSdk?.takeIf { it.sdkType is PythonSdkType }
+            state.consoleBuilder = PyDebugConsoleBuilder(environment.project, sdk)
```

Comment:
What happens if sdk is null? Does this still work or does it throw?

Reply:
It works, just seems to be stdlib only?